pkg/storage/internalstorage/register.go

```diff
@@ -90,9 +91,17 @@ func newLogger(cfg *Config) (logger.Interface, error) {
 			return nil, fmt.Errorf("mkdir log dir error: %w", err)
 		}
 
-		logWriter, err = os.OpenFile("/var/log/clusterpedia/internalstorage.log", os.O_APPEND|os.O_WRONLY|os.O_CREATE, 0644)
+		path := "/var/log/clusterpedia/internalstorage.log"
```

Comment:
The path should not be fixed. It should have a default value or be taken from yaml. In addition, consider what happens if the user doesn't want to keep the log and outputs it directly to the terminal.

Reply:
> The path should not be fixed. It should have a default value or be taken from yaml. 

I will make it flexible.
> In addition, consider what happens if the user doesn't want to keep the log and outputs it directly to the terminal.

We have the Stdout option now.